### PR TITLE
blog: add release 20.10

### PR DESCRIPTION
**Pull request by sduthil**

### CODE REVIEW

File: content/blog/sprint-review-2010.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+* **Asterisk**: Asterisk was upgraded to 17.5.1. See the [Asterisk release announcement](https://www.asterisk.org/downloads/asterisk-news/asterisk-1751-now-available)
+* **Performance**: We have finished removing the use of encryption for all internal HTTP communication between Wazo components. Encryption now only happens when Wazo is sending/receiving data from the outside world.
+* **Multi-tenancy**: We have improved the consistency of the database for multiple tenants. When a tenant is removed, most of the data related to this tenant are now removed.
+
+## Ongoing features
+
+* **SIP lines and trunks**: We are currently rewriting the SIP configuration API to match PJSIP options and be multi-tenant.
+
+For more details about the aforementioned topics, please see the roadmap linked below.
```
Comment (chartrandf) [inline content/blog/sprint-review-2010.md:23]: I don't see any link to a roadmap

File: content/blog/sprint-review-2010.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+* [Install Wazo Platform](/install)
+
+* [Upgrade Wazo and Wazo Platform](/uc-doc/upgrade/). Be sure to read the [breaking changes](/uc-doc/upgrade/upgrade_notes#20-10)
+
+Sources:
+
+* [Upgrade notes](/uc-doc/upgrade/upgrade_notes#20-10)
+
+* [Wazo Platform 20.10 Changelog](https://wazo-dev.atlassian.net/secure/ReleaseNote.jspa?projectId=10011&version=10101)
```
Comment (chartrandf) [inline content/blog/sprint-review-2010.md:37]: 20.09 change log

File: content/blog/sprint-review-2010.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Discussion
+
+Comments or questions in [this forum post](https://wazo-platform.discourse.group/t/blog-wazo-platform-20-10-released).
```
Comment (chartrandf) [inline content/blog/sprint-review-2010.md:41]: Seems to have a 404

File: content/blog/sprint-review-2010.md (hunk 1 above)
Comment (sduthil) [reply content/blog/sprint-review-2010.md]: I reworded to "changelog"

File: content/blog/sprint-review-2010.md (hunk 2 above)
Comment (sduthil) [reply content/blog/sprint-review-2010.md]: Fixed

**Comment by sduthil** [comment]
Thanks Francis for the review!

File: content/blog/sprint-review-2010.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+Status: published
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 20.10 release.
+
+## Technical features
+
+* **Asterisk**: Asterisk was upgraded to 17.5.1. See the [Asterisk release announcement](https://www.asterisk.org/downloads/asterisk-news/asterisk-1751-now-available)
+* **Performance**: We have finished removing the use of encryption for all internal HTTP communication between Wazo components. Encryption now only happens when Wazo is sending/receiving data from the outside world.
```
Comment (fblackburn1) [inline content/blog/sprint-review-2010.md:16]: We should remove this one:
* Already announced in 20.09
* See [my comment about upgrade note](https://github.com/wazo-platform/wazo-platform.org/pull/210/files#r457267942)

File: content/blog/sprint-review-2010.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+* **Multi-tenancy**: We have improved the consistency of the database for multiple tenants. When a tenant is removed, most of the data related to this tenant are now removed.
```
Comment (fblackburn1) [inline content/blog/sprint-review-2010.md:17]: Oh it's really not true. maybe less than 10% of data related to this tenant is removed. Maybe something like this:
```suggestion
* **Multi-tenancy**: We have fixed some consistency of the database for multiple tenants.
```

But IMO we should remove this one too

Comment (pc-m) [reply content/blog/sprint-review-2010.md:17]: Or be more specific.
`Users will now get deleted from the database after the tenant gets deleted.`

Comment (sduthil) [reply content/blog/sprint-review-2010.md]: I added Pascal's suggestion. Thanks!

File: content/blog/sprint-review-2010.md (hunk 4 above)
Comment (sduthil) [reply content/blog/sprint-review-2010.md]: Fixed. Thank you!

**Pull request merged**